hw/ip/clkmgr/data/clkmgr.hjson.tpl

```diff
@@ -9,8 +9,11 @@
   scan: "true",
   clocking: [
     {clock: "clk_i", reset: "rst_ni", primary: true},
-% for rst in clocks.reset_signals():
-    {reset: "${rst}"},
+% for src in clocks.srcs.values():
+    {clock: "clk_${src.name}_i", reset: "rst_${src.name}_ni"},
+% endfor
+% for src in clocks.derived_srcs.values():
+    {clock: "clk_${src.name}_i", reset: "rst_${src.name}_ni", internal: true},
```

Comment:
Is `drived_srcs` comes from module internal?

Reply:
yes that's right. The derived sources are the internally derived clocks.